File: Kaynak Kodlar/BIL 341-5 (Java)/src/File.java
public class File {
	
	public String fileName ;
	public int fileSize ;
	public int fileBlocks ;
	
	public File(String fileName, int fileSize) {
		super();
		this.fileName = fileName;
		this.fileSize = fileSize;
	}

	public String getFileName() {
		return fileName;
	}

	public void setFileName(String fileName) {
		this.fileName = fileName;
	}

	public int getFileSize() {
		return fileSize;
	}

	public void setFileSize(int fileSize) {
		this.fileSize = fileSize;
	}

	public int getFileBlocks() {
		return fileBlocks;
	}

	public void setFileBlocks(int fileBlocks) {
		this.fileBlocks = fileBlocks;
	}
	
	
}

File: Kaynak Kodlar/BIL 235-1 (C++) (2007)/src/file.cpp
#include <iostream.h>
#include <cstring>
#include <cassert>
#include "file.h"
using namespace std;

file :: file(string Id,string name,int size,string type)
{
	FileName = name ;
	setFileSize(size);
	FileId = Id ;
	FileType = type;
}

void file :: setFileSize(int FileSize)
{
	this->FileSize=FileSize;
}

int file :: getFileSize()const
{
	return FileSize;
}

string file :: getFileName() const
{
	return FileName;
}	

string file :: getFileId() const
{
	return FileId;
}

string file :: getFileType() const
{
	return FileType;
}
